models/plan.js

```diff
@@ -353,7 +353,7 @@ async function getPlan(planId, userId) {
   try {
 
     // remove all notifications for the plan
-    checkPlanNotifications(planId, userId);
+    deletePlanNotifications(planId, userId);
```

Comment:
`await`?

Reply:
The reason for no await here is a bit similar to the reason there is no await for a notification creation when a user adds a comment to the plan.

I only really want to justify awaiting for the notification to clear if I want to do something in the getPlan() function based on if there is an error.  In this instance the deletePlanNotifications() function is being called since we are about to view a specific plan. The function will delete any notifications telling us about the page we are about to view.

However if a user goes to a page to view a specific plan I feel like not being able to view the page due to the notifications failing to clear is a worse user experience than having the notifications not automatically clear when we view a page that the notifications reference.

If you have any other thoughts about this functionality I am happy to discuss it further.

Reply:
Oh, so this is the same as the notification. My bad, I kinda forgot about that discussion. All clear.